lib/kafo/configuration.rb

```diff
@@ -264,7 +264,7 @@ def migrations_dir
 
     def parser_cache
       if app[:parser_cache_path]
-        @parser_cache ||= Kafo::ParserCacheReader.new_from_file(File.expand_path(app[:parser_cache_path]))
+        @parser_cache ||= Kafo::ParserCacheReader.new_from_file(app[:parser_cache_path])
```

Comment:
Added a line https://github.com/theforeman/kafo/pull/172/files#diff-3f80fc96ce636e776708977123694238R7  to expand each path.